app/components/profile_header/profile_header.html.erb

```diff
@@ -3,8 +3,10 @@
     <%= t('.main_heading', business_plan_name: organization.business_plan.name) %>
   <% end %>
 
-  <%= c 'heading', style: :gamma do %>
-    <%= t('.sub_heading.contact_person', contact_person_name: organization.contact_person.name, contact_person_email: organization.contact_person.email) %>
+  <% if organization.contact_person.present? %>
```

Comment:
I, too, give it a :+1: although I'd liked to see a little unit test.

Reply:
Ok, so I added a component spec for the ProfileHeader component where this conditional is.
I also decided I might as well add a test for each conditional in the component.
This lead me to realize I was misusing the Heading component being explicit and passing in the name of the tag I wanted to be used. See  [[a11y] Check that there are not duplicate h1 tags on pages #1812 ](https://github.com/tactilenews/100eyes/issues/1812)

I also introduced the idea of test id's to test the presence or absence of certain elements instead of testing the presence or absence of some text, which can be flaky. You can read more about this tactic at this blog post [Why Your Development Team Should Use data-testid Attributes](https://medium.com/@automationTest/why-your-development-team-should-use-data-testid-attributes-a83f1ca27ebb)

I will go ahead and merge this, but am open to any feedback you may want to give @roschaefer @soey